chat-commands.js

```diff
@@ -2450,37 +2450,37 @@ exports.commands = {
 	blacklist: function (target, room, user) {
 		if (!target) return this.parse('/help blacklist');
 		if (!this.canTalk()) return;
-		if (toId(target) === 'show') return this.errorReply("You're looking for /showbl");
+		if (toId(target) === 'show') return this.errorReply(`You're looking for /showbl`);
 
 		target = this.splitTarget(target);
 		const targetUser = this.targetUser;
-		if (!targetUser) return this.errorReply("User '" + this.targetUsername + "' not found.");
+		if (!targetUser) return this.errorReply(`User ${this.targetUsername} not found. If you are trying to blacklist an offline account use /blacklistname`);
```

Comment:
That's a command, not a suggestion!

Also this should be two lines.

Reply:
so 
`if (!targetUser) return this.errorReply(
			`User ${this.targetUsername} not found. If you are trying to blacklist an offline account use /blacklistname`
		);` ?

Reply:
... no
```
if (!targetUser) {
     this.errorReply(`User ${this.targetUsername} not found.`);
     this.errorReply(`If you are trying to blacklist an offline account, use /blacklistname`);
     return;
}
```